packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -224,14 +249,24 @@ export async function registerCheckpoints (
       }
       case 'appl': {
         txConfirmation = await deployer.waitForConfirmation(txn.txID());
-        const sscInfo: rtypes.SSCInfo = {
-          creator: encodeAddress(txn.from.publicKey),
-          txId: txn.txID(),
-          appID: txConfirmation['application-index'],
-          confirmedRound: txConfirmation['confirmed-round'],
-          timestamp: Math.round(+new Date() / 1000)
-        };
+        const key = deployer.getAppCheckpointKeyFromIndex(txn.appIndex);
+        if (key) {
+          const temp: rtypes.SSCInfo | undefined = deployer.getSSCfromCPKey(key);
+          if (txn.appOnComplete === 5 && temp) {
+            temp.deleted = true;
+            deployer.registerSSCInfo(key, temp);
+            break;
```

Comment:
Shall we `deployer.logTx()` before breaking?

Reply:
yes